ipwb/util.py

```diff
@@ -34,7 +34,7 @@
 (IPWBREPLAY_HOST, IPWBREPLAY_PORT) = IPWBREPLAY_ADDRESS.split(':')
 IPWBREPLAY_PORT = int(IPWBREPLAY_PORT)
 
-INDEX_FILE = 'samples/indexes/salam-home.cdxj'
+INDEX_FILE = os.path.sep.join(['samples', 'indexes', 'salam-home.cdxj'])
```

Comment:
```suggestion
INDEX_FILE = os.path.join('samples', 'indexes', 'salam-home.cdxj')
```

Reply:
Sometimes, @ibnesayeed, I think that project progress can be made when I propose a solution and you make it more streamlined. As a caveat, prior to this PR, I was unaware of the existence of `os.path.sep.join`.

Reply:
The `os.path.sep` returns an OS-dependent string. `join` here is the usual `join` function we use on any string. It is the `os.path.join` function that is the different one.